draft-ietf-httpbis-encryption-encoding.md

```diff
@@ -598,6 +629,27 @@ added for presentation purposes only.
 ~~~
 
 
+## Diffie-Hellman with Authentication Secret
+
+This example shows the same receiver key pair from {{ex-dh}}, but with a shared
+authentication secret of "R29vIGdvbyBnJyBqb29iIQ".
+
+~~~
+HTTP/1.1 200 OK
+Content-Length: 33
+Content-Encoding: aesgcm
+Encryption: keyid="dhkey"; salt="lngarbyKfMoi9Z75xYXmkg"
+Crypto-Key: keyid="dhkey";
+                dh="BNoRDbb84JGm8g5Z5CFxurSqsXWJ11ItfXEWYVLE85Y7
+                    CYkDjXsIEc4aqxYaQ1G8BqkXCJ6DPpDrWtdWj_mugHU"
+
+QPsWCEH-8WqiXl7Ez23A6u4rwH2hFBifCybQQz-SxsjF
```

Comment:
Ahh, I get the same answer that you do.  Strange.  I must have I copied the wrong value in somehow.  Thanks for checking!  I'll change the ciphertext.